backend/program/scrapers/jackett.py

```diff
@@ -32,6 +37,7 @@ def validate(self) -> bool:
             self.api_key = self.settings.api_key
             try:
                 url = f"{self.settings.url}/api/v2.0/indexers/!status:failing,test:passed/results/torznab?apikey={self.api_key}&cat=2000&t=movie&q=test"
+                self.ranking_model = models.get(self.rank_profile)

```

Comment:
Will this keep the reference to the settings_manager object? ie. if you change settings on the fly what happens?

Reply:
@dreulavelle